controls/radtimeline/getting-started.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Getting Started
+page_title: Getting Started
+description: Getting Started
+slug: radtimeline-getting-started
+tags: getting,started
+published: True
+position: 1
+---
+
+# Getting Started
+
+This topic will walk you through the creating of  __RadTimeline__.    
+
+>In order to use __RadTimeline__ in your project you have to add references to the following two assemblies:
```

Comment:
Can we move the assemblies list into an "Assembly References" section as the one in the RadGridView Getting Started article?